.github/pull_request_template.md

```diff
@@ -1,16 +1,25 @@
-# Description
+## Description
+<!-- Please include a summary of what this PR is changing and why these changes are needed. -->
 
-Please include a summary of the changes and the related issue.
+Fixes # <!-- issue number, if applicable -->
 
-Fixes # (issue)
+## Testing Instructions
+<!-- Please include step by step instructions on how to test this PR. -->
+<!-- 1. Tap on the Filters tab -->
+<!-- 2. Tap on a filter -->
+<!-- 3. etc. -->
 
-# Checklist
+## Screenshots or Screencast 
+<!-- if applicable -->
 
-- [ ] Should this change be included in the release notes? If yes, please add a line in CHANGELOG.md
-- [ ] Have you tested in landscape?
-- [ ] Have you tested accessibility with TalkBack?
-- [ ] Have you tested in different themes?
-- [ ] Does the change work with a large display font?
-- [ ] Are all the strings localized?
```

Comment:
I dropped the localized strings checklist items. I'm not sure the benefit from this is worth the noise of another checklist item since this will almost certainly be caught during PR review and it's easy to fix. Let me know if you think we should keep it.

Reply:
Instead of removing "Are all the strings localized?" maybe it could say something like:
```
- [ ] Are all the strings which need to be localized in modules/services/localization/src/main/res/values/strings.xml?
```
Or something more friendly. I [noticed a PR](https://github.com/Automattic/pocket-casts-android/pull/427/files) with a string to be localized in another module.

Reply:
Adding the location where the localized strings should go is a good idea. Added in [bdf4159](https://github.com/Automattic/pocket-casts-android/pull/481/commits/bdf415920b13db1efbddd94f64a957714fdf2f0a).